data/card_catalogue/upgradeable-1.json

```diff
@@ -234,7 +74,7 @@
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Turn off the power in the current Cube. Draw 3 copies of Fork Lightning.",
+      "rules": "Cause a complete blackout in a 100 meter radius. Draw 3 copies of Fork Lightning",
```

Comment:
Removing the direct lore reference, but keeping the general idea

Reply:
For the sake of lore (and since it makes no gameplay difference) I think we should crank that radius to 100 kilometers

Reply:
Oh fuck, alright let's go